AprilSylph/XKit-Rewritten
# Show Originals: Fix blog subscriptions tab

### Description


Resolves #818.

As per the linked issue, there's no `data-which` property on the blog subscriptions tab https://www.tumblr.com/dashboard/blog_subs. This seemed like the minimum total person-hours to fix Show Originals not appearing on it, though I guess if it's technically a redpop bug it's best that it gets fixed at some point to avoid causing further problems later.

### Testing steps


Go to https://www.tumblr.com/dashboard/blog_subs. There should be a Show Originals toggle on it.
